### DISCUSSION TITLE
collagraph Qt Style Sheets

### DISCUSSION THREAD

**Discussion by jamesdbrock** [fork-tongue/collagraph#106] [General]
Is there a way to set `QWidget` styles with Qt Style Sheet syntax? https://doc.qt.io/qtforpython-6/overviews/stylesheet.html#qt-style-sheets

**Comment by berendkleinhaneveld** [comment]
That should be possible by setting the 'property' `style-sheet` on a widget: that should call the `setStyleSheet` method.
